### PR TITLE
Remove out of scope changes

### PR DESCRIPTION
**Pull request by Monkeychip**
**TLDR** 
* This pr removes any out of scope changes from my side branch. This includes changing hardcode values to vars and adding extra spaces between class definitions.
* Small changes to get tests passing.

Jordan had left a comment on my[ last PR](https://github.com/hashicorp/vault/pull/20196/files#diff-e4a6c8154565bc0b278bb723063e63dbe12a4a035070225aa3770e96a8ae0100) about changing hardcode px or rem values to vars. Ex: `padding: 1rem` to `padding: $size-6`. After looking at my sidebranch diff against main I saw how doing this extra work had done a number on the already large file count. If we decide as a team that it's worth going through all previously hard-coded rem and px and replacing them with size vars when applicable we can do that. However, to keep the scope of this PR as focused as possible I removed all these changes I made over the course of working on this project.

### CODE REVIEW

File: ui/app/styles/components/tabs.scss [hunk 1]
```diff
@@ -62,12 +62,12 @@
     }
```
Comment (Monkeychip) [inline ui/app/styles/components/tabs.scss:63]: alphabetized css block

**Review by zofskeez — APPROVED** [review]
👏 thanks for following up on this! 🚀

**Comment by Monkeychip** [comment]
> 👏 thanks for following up on this! 🚀

Thanks for the discussion!
